### DISCUSSION TITLE
Issuies link retirect to web instead of Github mobile application.

### DISCUSSION THREAD

**Discussion by meokullu** [community/community#71591] [Mobile]
### Select Topic Area

Bug

### Body

Hi. I just noticed that issues link is not opening in app instead opening as a web link.

https://github.com/meokullu/HelpConsole/issues/7

Take care.

**Comment by mattruggio** [comment]
Hey @meokullu, I want to make sure I understand this a bit more. Could you please provide us what device make/model, device OS version, GitHub app. version are you currently using? 

Also, where is this link being attempted to be opened from? For example: is this being clicked on within mobile Safari or Firefox applications or possibly the built-in mobile mail app?

**Reply by meokullu** [reply]
Hi @mattruggio 

Apple iPhone iOS 17.1
GitHub iOS 1.130.0.113105320
I shared link on Tweet and here is tweet
https://twitter.com/enesokullu/status/1716484332754206724
![image](https://github.com/community/community/assets/4971757/73835aa0-8589-46f8-b95f-6ad0d8627f7a)
![image](https://github.com/community/community/assets/4971757/c74d44e0-f9e7-4e3b-8b4f-ca57d9b9bf0e)
![image](https://github.com/community/community/assets/4971757/11b151ca-b440-41a4-a074-cfb59ea48061)

I am sorry of too many edits. I am not in my best mood now and I wanted to deliver informations to you as soon as possible.

Take care.
